jerry-core/parser/js/lexer.cpp

```diff
@@ -1482,6 +1478,7 @@ lexer_next_token (void)
   if (!is_empty (saved_token))
   {
     sent_token = saved_token;
+    prev_non_lf_token = sent_token;
```

Comment:
`prev_non_lf_token` should be tracked more carefully. It seems, that it should be changed in `lexer_seek` and `lexer_save_token` functions.
For example, the following code:
```
{for (;false;/abc/.exec("abc")) (5)}
```
fails with syntax error due to `prev_non_lf_token` equals to `TOK_CLOSE_BRACE` after call to `lexer_seek` (parser.cpp:1920).